From 3a447704e56902115638917c4072c8e50bea851e Mon Sep 17 00:00:00 2001
Message-Id: <3a447704e56902115638917c4072c8e50bea851e.1529072479.git.marcel.ziswiler@toradex.com>
In-Reply-To: <fb4764e8eb658d35e8fc62ae79c77e1f6e2b0ef3.1529072479.git.marcel.ziswiler@toradex.com>
References: <fb4764e8eb658d35e8fc62ae79c77e1f6e2b0ef3.1529072479.git.marcel.ziswiler@toradex.com>
From: Marcel Ziswiler <marcel.ziswiler@toradex.com>
Date: Sun, 18 Jun 2017 01:23:26 +0200
Subject: [PATCH 06/27] apalis/colibri_t30, apalis-tk1: snapd squashfs
 configuration

Prepare for snapd integration.

Signed-off-by: Marcel Ziswiler <marcel.ziswiler@toradex.com>
Acked-by: Dominik Sliwa <dominik.sliwa@toradex.com>
(downstream tegra commit a24d47d56d9f5a6ed1ec3a42c6d9ccf5748bc2ba)
(downstream toradex_tk1_l4t_r21.5 commit
f328e55a793497870c533eac0ac21cb735a82e6d)
---
 arch/arm/configs/tegra_defconfig | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/tegra_defconfig b/arch/arm/configs/tegra_defconfig
index cb97ea2db1e0..cfe997c617fc 100644
--- a/arch/arm/configs/tegra_defconfig
+++ b/arch/arm/configs/tegra_defconfig
@@ -295,7 +295,12 @@ CONFIG_AUTOFS4_FS=y
 CONFIG_VFAT_FS=y
 CONFIG_TMPFS=y
 CONFIG_TMPFS_POSIX_ACL=y
-CONFIG_SQUASHFS=y
+CONFIG_SQUASHFS=m
+CONFIG_SQUASHFS_FILE_DIRECT=y
+CONFIG_SQUASHFS_DECOMP_MULTI_PERCPU=y
+CONFIG_SQUASHFS_XATTR=y
+CONFIG_SQUASHFS_ZLIB=y
+CONFIG_SQUASHFS_LZ4=y
 CONFIG_SQUASHFS_LZO=y
 CONFIG_SQUASHFS_XZ=y
 CONFIG_NFS_FS=y
-- 
2.14.4

